src/file/document/body/section-properties/doc-grid/doc-grid.ts

```diff
@@ -1,13 +0,0 @@
-import { XmlComponent } from "file/xml-components";
-import { DocGridAttributes } from "./doc-grid-attributes";
-
-export class DocumentGrid extends XmlComponent {
```

Comment:
A lot of deleted files

Waiting to see where these get added in

Reply:
A lot of things were moved around, a lot of them were duplicates of the same underlying types and were deleted.